src/types/ISwap.ts

```diff
@@ -8,22 +8,12 @@ export type Quote = {
 };
 
 export type Token = {
-    symbol: string;
-    name: string;
     address: string;
-    standard: string;
-
-    // TODO: should this be always included or make it optional
-    // additional info
-    id?: bigint;
-    volumeUSD1d?: number;
-    volumeUSD7d?: number;
-    totalVolumeUSD?: number;
-    volumeUSD?: number;
-    feesUSD?: number;
-    priceUSDChange?: number;
-    txCount?: bigint;
-    priceUSD?: number;
+    // icpswap props
+    name?: string;
```

Comment:
do we really need `name` and `symbol` when making request for `quote` or `swap`? those seem to be used in other places though

Should there be 2 different types `Token` and `TokenView` maybe?